apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -488,6 +565,44 @@ export default Vue.extend({
             this.copyCell(cell)
           }
         },
+        {
+          label: `<x-menuitem><x-label>Edit row as JSON</x-label></x-menuitem>`,
+          disabled: (cell: Tabulator.CellComponent) => !this.editable && !this.insertionCellCheck(cell),
+          action: (_e, cell) => {
+            if (!this.isPrimaryKey(cell.getField()) && this.primaryKeys.length >= 1) {
+              const column = cell.getField()
+              const row = cell.getRow()
+              const valueCell = row.getCell(column);
+              const value = valueCell.getValue();
+              let parsed: null | Record<string,unknown> = null
+
+              try {
+                parsed = JSON.parse(value)
+
+              } catch (e) {
+                console.log("Invalid JSON", e)
```

Comment:
If you think it's important that this is logged, then use `log.debug` or `log.error` instead of `console.log`. That way it won't pollute the browser console. Otherwise just remove it

Reply:
I agree with `log.error`